lib/features/study_circle_details/models/contact_section_data.dart

```diff
@@ -18,28 +17,11 @@ class ContactSectionData {
   ContactSectionData({
     required this.text,
     required this.url,
-    required this.type,
-  }) {
-    if (url != null) {
-      _invalidateUrl();
-      _determineIcon();
-    }
-  }
+  }) : iconUrl = _determineIcon(url);
 
-  void _invalidateUrl() {
-    String prefix = switch (type) {
-      Enum$ENUM_COMPONENTINFOINFO_TYPE.Email => 'mailto:',
-      Enum$ENUM_COMPONENTINFOINFO_TYPE.PhoneNumber => 'tel:',
-      Enum$ENUM_COMPONENTINFOINFO_TYPE.Website => 'https://',
-      _ => ""
-    };
-    if (url?.startsWith(prefix) == false) {
-      Logger().w("Following url: $url is not properly formatted.");
-      url = prefix + url!;
-    }
+  static String _determineIcon(String? url) {
+    if (url == null) return DetailsScreenConfig.defaultIconUrl;
```

Comment:
Here as well, we could use tenary conditional operator (same as before) but still, not forcing anything, just matter of preference :)